content/guides/creating-test-money.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Creating Test Money
```

Comment:
We should delete these pages from the legacy guides as well :)

Reply:
I have removed the pages from the legacy guides.

Reply:
Nice!